### PR TITLE
Edit deck

### PR DESCRIPTION
**Pull request by roseandlily33**
Changed some req.body.id to req.session.user_id, 
Delete, and create should be working 
Still working on the edit page
Changed some of the variables for routers pulling different information
Changed some of the logic in the new-card.js it wasn't giving the right card numbers when I did the console.log and was starting at 6 and now it starts at 0 and goes up.
Ignore the update.js file that's just some boiler plate code that I copied pasted for the edit page and am working on. 
I added the cards per day option on the create and edit files. 

You should be able to create a profile, create a deck, see the decks that only you've created and can click on the edit, and can delete the deck, nothing inside of the values for the cards yet.

### CODE REVIEW

File: models/Card.js
Comment (Pilotguide9897) [inline models/Card.js:1]: I think that we are just removing this.

File: models/User.js
Comment (Pilotguide9897) [inline models/User.js:1]: Lorne has already added the is unique validator, so I will just keep it the way it is when resolving the merge conflict.

**Review by Pilotguide9897 — DISMISSED** [review]
Good Job Victoria. Things look good to me. The shape of the app is really starting to come together!

**Review by Pilotguide9897 — APPROVED** [review]

File: models/Card.js
Comment (roseandlily33) [reply models/Card.js:1]: When adding a new deck it was complaining that there was no default value set for this one for some reason. If it comes up again, we can add it, if you took it out.

**Pull request merged**